src/objectTranslations/npe03__Recurring_Donation__c-es.objectTranslation

```diff
@@ -160,58 +169,71 @@
+            <translation>ACH/EFT</translation>
         </picklistValues>
         <picklistValues>
             <masterLabel>Check</masterLabel>
-            <translation><!-- Check --></translation>
+            <translation>Cheque</translation>
         </picklistValues>
         <picklistValues>
             <masterLabel>Credit Card</masterLabel>
-            <translation><!-- Credit Card --></translation>
+            <translation>Tarjeta de crédito</translation>
         </picklistValues>
     </fields>
     <fields>
-        <help><!-- Default value is Open, meaning a recurring donation with no end date. --></help>
-        <label><!-- Recurring Type --></label>
+        <help>El valor predeterminado es Abierto, lo que significa una donación recurrente si fecha de finalización.</help>
+        <label>Tipo de recurrencia</label>
         <name>RecurringType__c</name>
+        <picklistValues>
+            <masterLabel>Fixed</masterLabel>
+            <translation>Fijo</translation>
+        </picklistValues>
         <picklistValues>
             <masterLabel>Open</masterLabel>
-            <translation><!-- Open --></translation>
+            <translation>Abierto</translation>
         </picklistValues>
     </fields>
     <fields>
-        <help><!-- The date of the first installment Opportunity for this Recurring Donation. Changes made to this field after the schedule has started will not impact future installment Opportunities or the schedule. The default is the current date. --></help>
-        <label><!-- Start Date --></label>
+        <help>La fecha en la que entra en vigor esa información de programación nueva o actualizada (Importe, Día del mes, etc.) para esta Donación recurrente. El valor predeterminado es la fecha actual.</help>
+        <label>Fecha efectiva</label>
         <name>StartDate__c</name>
     </fields>
     <fields>
-        <help><!-- Indicates if this Recurring Donation is actively in use, temporarily suspended, or closed. --></help>
-        <label><!-- Status --></label>
+        <help>Indica si esta Donación recurrente está en uso activamente, suspendida temporalmente o cerrada.</help>
+        <label>Estado</label>
         <name>Status__c</name>
         <picklistValues>
             <masterLabel>Active</masterLabel>
-            <translation><!-- Active --></translation>
+            <translation>Activo</translation>
         </picklistValues>
         <picklistValues>
             <masterLabel>Closed</masterLabel>
-            <translation><!-- Closed --></translation>
+            <translation>Cerrado</translation>
         </picklistValues>
         <picklistValues>
             <masterLabel>Lapsed</masterLabel>
-            <translation><!-- Lapsed --></translation>
+            <translation>Transcurrido</translation>
         </picklistValues>
     </fields>
+    <quickActions>
+        <label>Cerrar donación recurrente</label>
+        <name>Close_Recurring_Donation</name>
```

Comment:
@GloryChoe Looks like this is missing the translation for the SuccessMessage attribute of the Quick Action: https://github.com/SalesforceFoundation/NPSP/blob/master/src/quickActions/npe03__Recurring_Donation__c.Close_Recurring_Donation.quickAction#L24

This should be translatable and be included here as another translated attribute.
CC @mpusto